### DISCUSSION TITLE
Configuration with env variables

### DISCUSSION THREAD

**Discussion by static-moonlight** [versity/versitygw#424] [Ideas]
How do you feel about the idea to support environment variables for configuration? I checked the code. The CLI framework you're using would support it. All it takes is to define them and put them in. Maybe even with a general prefix, like `VGW_` (that seems to be some kind of "best practice", used by many applications)?! I could help you out, if you want ;)

**Comment by benmcclelland** [comment]
@static-moonlight that sounds like a good idea to me. That would be great if you want to create a PR for this. VGW_ looks like a good env prefix to me.

**Comment by static-moonlight** [comment]
Pull request is open #427
